packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionInvoke.ts

```diff
@@ -41,3 +44,35 @@ export async function forceFunctionInvoke(sourceUri: Uri) {
   );
   await commandlet.run();
 }
+
+export async function forceFunctionDebugInvoke(sourceUri: Uri) {
+  const localRoot = FunctionService.getFunctionDir(sourceUri.fsPath);
+  if (!localRoot) {
```

Comment:
Trying to understand why this check is needed. Wouldn't "Debug Send Request" only appear for valid payload files inside functions directory?

Reply:
Yeah, "Debug Send Request" is appearing for valid files under `functions` dir. This check is specifically checking for `function.toml`, where "start function" uses for locating the function directory. If you removed the toml file in a valid function dir, this message appears.